ledger/participant-state/kvutils/app/src/test/scala/com/daml/ledger/participant/state/kvutils/app/ConfigSpec.scala

```diff
@@ -3,31 +3,66 @@
 
 package com.daml.ledger.participant.state.kvutils.app
 
+import com.daml.ledger.participant.state.v1.ParticipantId
 import org.scalatest.{FlatSpec, Matchers, OptionValues}
 import scopt.OptionParser
 
 final class ConfigSpec extends FlatSpec with Matchers with OptionValues {
 
-  private val DumpIndexMetadataCommand = "dump-index-metadata"
-  private val ConfigParser: Seq[String] => Option[Config[Unit]] =
-    Config.parse("Test", (_: OptionParser[Config[Unit]]) => (), (), _)
+  private val dumpIndexMetadataCommand = "dump-index-metadata"
+  private val participantOption = "--participant"
+  private val participantId: ParticipantId = ParticipantId.assertFromString("dummy-participant")
+  private val fixedParticipantSubOptions = s"participant-id=$participantId,port=123"
```

Comment:
I'd suggest naming this as `mandatoryParticipantOptions` or `mandatoryParticipantConfig` or similar to hint that these must be provided.